### PR TITLE
Use correct ids for nas and server metrics

### PR DESCRIPTION
**Pull request by 0xAX**

### CODE REVIEW

File: test/eradius_config_SUITE.erl [hunk 1]
```diff
@@ -25,6 +25,7 @@
 all() -> [config_1, config_2, config_with_ranges, log_test].
 
 init_per_suite(Config) ->
+    io:format("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA ~n"),
```
Comment (liveforeverx) [inline test/eradius_config_SUITE.erl]: Do not forget to remove io:format

File: src/eradius_metrics.erl [hunk 2]
```diff
@@ -46,9 +46,11 @@ server_metric_name(IP, Port, Metric, Type) ->
...
 	    eradius_metrics:get_metric_name(IP, Port, Metric, Type);
 	_ ->
-	    FormatNasName = [IP, <<":">>, integer_to_binary(Port)],
-	    Key = binary_to_atom(erlang:iolist_to_binary(FormatNasName), utf8),
-	    [eradius, Type, Key, Metric]
+	    % TODO
+	    %FormatNasName = [IP, <<":">>, integer_to_binary(Port)],
+	    %Key = binary_to_atom(erlang:iolist_to_binary(FormatNasName), utf8),
+	    % IP - NasID
+	    [eradius, Type, IP, Metric]
```
Comment (surik) [inline src/eradius_metrics.erl]: IP or NasID?

File: src/eradius_metrics.erl [hunk 3]
```diff
@@ -16,9 +16,9 @@ subscribe_server(IP, Port, SubscriptionType) ->
 	case lists:member(SubscriptionType, EnabledMetrics) of
 	    true ->
 		lists:foreach(fun(Metric) ->
-		    Name = server_metric_name(IP, Port, Metric, SubscriptionType),
-		    exometer_report:subscribe(Reporter, Name, MetricType, 1000, [{SubscriptionType, {from_name, 3}}], true)
-		end, MetricsList);
+				      Name = server_metric_name(IP, Port, Metric, SubscriptionType),
+				      exometer_report:subscribe(Reporter, Name, MetricType, 1000, [{SubscriptionType, {from_name, 3}}], true)
+			      end, MetricsList);
```
Comment (surik) [inline src/eradius_metrics.erl]: tabs?

File: src/eradius_server.erl [hunk 4]
```diff
@@ -68,18 +68,20 @@
...
 
 %% @private
--spec start_link(inet:ip4_address(), port_number()) -> {ok, pid()} | {error, term()}.
-start_link(IP = {A,B,C,D}, Port) ->
+-spec start_link(atom(), inet:ip4_address(), port_number()) -> {ok, pid()} | {error, term()}.
+start_link(ServerName, IP = {A,B,C,D}, Port) ->
     Name = list_to_atom(lists:flatten(io_lib:format("eradius_server_~b.~b.~b.~b:~b", [A,B,C,D,Port]))),
     io:format("Name: ~p~n", [Name]),
-    gen_server:start_link({local, Name}, ?MODULE, {IP, Port}, []).
+    eradius_metrics:subscribe_server(atom_to_binary(ServerName, utf8), server),
```
Comment (surik) [inline src/eradius_server.erl]: why do you transform ServerName to binary? can we use it as atom?

**Pull request merged**